deployment/network-operator/values.yaml

```diff
@@ -16,6 +16,20 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+admissionController:
```

Comment:
IMO, it should be under `operator` [1] section. @adrianchiris @ykulazhenkov  what do you think?

[1] https://github.com/Mellanox/network-operator/blob/a0274df5ee4038db3f6ed3badeadd0cef4744771/deployment/network-operator/values.yaml#L114

Reply:
I agree with Ivan